README.md

```diff
@@ -21,6 +21,13 @@ You can read about Markdown [here](https://guides.github.com/features/mastering-
 or [other](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
 [places](http://www.mkdocs.org/user-guide/writing-your-docs/).
 
+Code examples are included from source using 
+[`mkdocs-codeinclude-plugin`][codeinclude-plugin] to ensure they are
+always up-to-date and correct.
+
+<!-- todo: Replace the link to our fork with the upstream link once our PRs are integrated. -->
```

Comment:
add here PR links to do not search them in future

Reply:
There are about 4 of them: https://github.com/rnorth/mkdocs-codeinclude-plugin/pulls